### PR TITLE
Microsoft.DesktopVirtualization 2024-04-08-preview

### PR DESCRIPTION
**Pull request by almat-msft**
# ARM (Control Plane) API Specification Update Pull Request 

> [!TIP]
> Overwhelmed by all this guidance? See the `Getting help` section at the bottom of this PR description.

## PR review workflow diagram

Please understand this diagram before proceeding. It explains how to get your PR approved & merged.

spec_pr_review_workflow_diagram

## Purpose of this PR

What's the purpose of this PR? Check the specific option that applies. This is **mandatory**!

  - [ ] New resource provider.
  - [x] New API version for an existing resource provider. (If API spec is not defined in TypeSpec, the PR should have been created in adherence to [OpenAPI specs PR creation guidance](https://aka.ms/azsdkdocs/createopenapispec)).
  - [ ] Update existing version for a new feature. (This is applicable only when you are revising a private preview API version.)
  - [ ] Update existing version to fix OpenAPI spec quality issues in S360.
  - [ ] Other, please clarify:
    - _edit this with your clarification_

## Due diligence checklist

To merge this PR, you **must** go through the following checklist and confirm you understood 
and followed the instructions by checking all the boxes:

- [x] I confirm this PR is modifying Azure Resource Manager (ARM) related specifications, and not data plane related specifications.
- [x] I have reviewed following [Resource Provider guidelines](https://aka.ms/rpguidelines), including
  [ARM resource provider contract](https://aka.ms/azurerpc) and
  [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md) (estimated time: 4 hours).  
  I understand this is required before I can proceed to the diagram Step 2, "ARM API changes review", for this PR.

## Additional information

<details>
<summary> Viewing API changes</summary>

For convenient view of the API changes made by this PR, refer to the URLs provided in the table 
in the `Generated ApiView` comment added to this PR. You can use ApiView to show API versions diff. 

</details>
<details>
<summary>Suppressing failures</summary>

If one or multiple validation error/warning suppression(s) is detected in your PR, please follow the 
[suppressions guide](https://aka.ms/azsdk/pr-suppressions) to get approval.

</details>

## Getting help

- First, please carefully read through this PR description, from top to bottom. Please fill out the `Purpose of this PR` and `Due diligence checklist`.
- To understand what you must do next to merge this PR, see the `Next Steps to Merge` comment. It will appear within few minutes of submitting this PR and will continue to be up-to-date with current PR state.
- For guidance on fixing this PR CI check failures, see the hyperlinks provided in given failure 
  and https://aka.ms/ci-fix.
- For help with ARM review (PR workflow diagram Step 2), see https://aka.ms/azsdk/pr-arm-review.
- If the PR CI checks appear to be stuck in `queued` state, please add a comment with contents `/azp run`.
  This should result in a new comment denoting a `PR validation pipeline` has started and the checks should be updated after few minutes.
- If the help provided by the previous points is not enough, post to https://aka.ms/azsdk/support/specreview-channel and link to this PR.

### CODE REVIEW

File: specification/common-types/resource-management/v6/types.json [hunk 1]
```diff
@@ -0,0 +1,748 @@
+{
```
Comment (abatishchev) [inline specification/common-types/resource-management/v6/types.json:1]: Please extract adding a new common type into a separate PR

**Review by abatishchev — DISMISSED** [review]

File: specification/common-types/resource-management/v6/types.json (hunk 1 above)
Comment (almat-msft) [reply specification/common-types/resource-management/v6/types.json]: ✅ published #29155

File: specification/desktopvirtualization/resource-manager/readme.python.md [hunk 2]
```diff
@@ -11,6 +11,8 @@ package-name: azure-mgmt-desktopvirtualization
 namespace: azure.mgmt.desktopvirtualization
 package-version: 1.0.0b1
 clear-output-folder: true
+modelerfour:
+  lenient-model-deduplication: true
```
Comment (msyyc) [inline specification/desktopvirtualization/resource-manager/readme.python.md:15]: Why do you add the configuration here?

Comment (almat-msft) [reply specification/desktopvirtualization/resource-manager/readme.python.md]: We needed it initially when we added a new identity property. Now we have removed the changes so these should not be needed anymore.

File: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/examples/SessionHostConfigurationsOperationStatus_Get.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (razvanbadea-msft) [inline specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/examples/SessionHostConfigurationsOperationStatus_Get.json:1]: this example file is not referenced.

**Comment by razvanbadea-msft** [comment]
>           "200": {

Being an async post this should be a 202 Accepted [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#202-accepted-and-location-headers](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#202-accepted-and-location-headers)

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:2807 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>           "description": "Default: AVD-wide settings are used to determine connection availability, Enabled: UDP will attempt this connection type when making connections. This means that this connection is possible, but is not guaranteed, as there are other factors that may prevent this connection type, Disabled: UDP will not attempt this connection type when making connections",

you should expand acronyms in descriptions

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:5595 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>         "systemData": {

systemData is added in the base definition in v5 common-types - [https://github.com/Azure/azure-rest-api-specs/blob/680f81b6ef2a6f0216aaa77e3ef5a2cb680b00bb/specification/common-types/resource-management/v5/types.json#L30](https://github.com/Azure/azure-rest-api-specs/blob/680f81b6ef2a6f0216aaa77e3ef5a2cb680b00bb/specification/common-types/resource-management/v5/types.json#L30)

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:5888 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>         "deleteOriginalVm": {

Replace boolean/switch properties with a more meaningful enum whenever possible. A boolean will forever have two valid values (true or false). Enums are always a more flexible and future proof option because they allow additional values to be added in the future in a non-breaking way, e.g. [Enabled, Disabled, Suspended, Deallocated].

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:5981 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>           "description": "The cancel message sent to the user on the session host. This is can only be specified if the action is 'Cancel'.",

maybe you need to rephrase it

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:6050 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>         "status": {

Based on the description this should be converted to an enum

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:6110 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>     "SessionHostManagementOperationType": {

for consistency add a description

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:6165 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>         "systemData": {

systemData is already present in common-types/resource-management/v5/types.json#/definitions/Resource

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:6324 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>           "additionalProperties": {

Use of additionalProperties is not allowed for properties owned by the service. The only time its ok to use it is when the properties are pass thru (user defined) and not subject to any validations. Please make this an array or provide an explanation for why you need this.

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:6361 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>           "description": "The id of the size of a virtual machine connected to a hostpool."

What format does this property has?

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:6393 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>         "customConfigurationScriptUrl": {

you can add uri format to it

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:6399 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>         "systemData": {

systemData is already present in common-types/resource-management/v5/types.json#/definitions/Resource

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:6456 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>         "vmSizeId": {

What format does this property has?   

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:6538 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>         "customConfigurationScriptUrl": {

you cand add uri format to it

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:6546 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>         "customConfigurationScriptUrl": {

you can add uri format to it

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:6632 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>         "type": {

You should rename this as it conflicts with the ARM top level property names

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:6666 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>         "type": {

You should rename this as it conflicts with the ARM top level property names

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:6735 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>           "type": "string",

Are there any format or restrictions that can be applied to this properties?

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:6818 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>           "description": "The ou path."

you should expand acronyms in descriptions 

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:6946 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>           "description": "The mdm guid."

you should expand acronyms in descriptions

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:6973 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>         "usernameKeyVaultSecretUri": {

you should add the uri format 

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:6985 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>         "usernameKeyVaultSecretUri": {

you should add the uri format 

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:6999 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>         "passwordKeyVaultSecretUri": {

you should add the uri format 

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:7003 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>         "passwordKeyVaultSecretUri": {

you should add the uri format 

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:6989 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>         "subnetId": {

is there any format that can be applied?

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:7016 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>           "description": "The resource ID of the security group. Any allowable/open ports should be specified in the NSG."

expand acronym

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:7022 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>         "type": {

You should rename this as it conflicts with the ARM top level property names

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:7044 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>         "secureBootEnabled": {

Replace boolean/switch properties with a more meaningful enum whenever possible. 

A boolean will forever have two valid values (true or false). Enums are always a more flexible and future proof option because they allow additional values to be added in the future in a non-breaking way, e.g. [Enabled, Disabled, Suspended, Deallocated].

For example two switches "isTypeA" and "isTypeB" should be replaced with one enum "type": [A, B, DefaultType]. 

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:7072 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>         "storageUri": {

you can add the uri format to it

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:7130 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>         "systemData": {

systemData is already present in the TrackedResource common type

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:7604 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>           "additionalProperties": {

Use of additionalProperties is not allowed for properties owned by the service. The only time its ok to use it is when the properties are pass thru (user defined) and not subject to any validations. Please make this an array or provide an explanation for why you need this.

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:7626 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>         "packageLookbackUrl": {

you should add the uri format    

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:7691 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>         "keyVaultURL": {

you should add the uri format    

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:7679 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>         "packageLookbackUrl": {

you should add the uri format    

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:7725 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>           "description": "SessionHostConfiguration version reference at the time the update is initiated, in the format of date time.",

you can add more details about the format if it is important

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:8609 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by razvanbadea-msft** [comment]
>       "description": "The name of the App Attach package arm object",

you should use Azure here

---
Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:9899 in 0110f47. [](commit_id = 0110f4755725de585b210001d1da98d428423eb0, deletion_comment = False)

**Comment by almat-msft** [comment]
> > ```
> >   "description": "The name of the App Attach package arm object",
> > ```
> 
> you should use Azure here
> 
> Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:9899 in [0110f47](https://github.com/Azure/azure-rest-api-specs/commit/0110f4755725de585b210001d1da98d428423eb0). [](commit_id = [0110f47](https://github.com/Azure/azure-rest-api-specs/commit/0110f4755725de585b210001d1da98d428423eb0), deletion_comment = False)

we use ARM everywhere else - won't this make it confusing?

**Comment by almat-msft** [comment]
> customConfigurationScriptUrl

this is not a top level property, it is nested under `sessionHostConfiguration.diskInfo.type` so it should not cause issues

**Comment by almat-msft** [comment]
> customConfigurationScriptUrl

this is not a top level property, it is nested under sessionHostConfiguration.diskInfo.type** so it should not cause issues

**Comment by almat-msft** [comment]
> > ```
> >     "type": {
> > ```
> 
> You should rename this as it conflicts with the ARM top level property names
> 
> Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:7044 in [0110f47](https://github.com/Azure/azure-rest-api-specs/commit/0110f4755725de585b210001d1da98d428423eb0). [](commit_id = [0110f47](https://github.com/Azure/azure-rest-api-specs/commit/0110f4755725de585b210001d1da98d428423eb0), deletion_comment = False)

This is used as part of `SessionHostConfiguration.diskInfo.type` meaning that it is not a top level property.

**Comment by almat-msft** [comment]
> > ```
> >     "type": {
> > ```
> 
> You should rename this as it conflicts with the ARM top level property names
> 
> Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:6735 in [0110f47](https://github.com/Azure/azure-rest-api-specs/commit/0110f4755725de585b210001d1da98d428423eb0). [](commit_id = [0110f47](https://github.com/Azure/azure-rest-api-specs/commit/0110f4755725de585b210001d1da98d428423eb0), deletion_comment = False)

This is part of `SessionHostConfiguration.imageInfo.type` which is not top level

**Comment by almat-msft** [comment]
> > ```
> >     "type": {
> > ```
> 
> You should rename this as it conflicts with the ARM top level property names
> 
> Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:6666 in [0110f47](https://github.com/Azure/azure-rest-api-specs/commit/0110f4755725de585b210001d1da98d428423eb0). [](commit_id = [0110f47](https://github.com/Azure/azure-rest-api-specs/commit/0110f4755725de585b210001d1da98d428423eb0), deletion_comment = False)

This is used as part of `.definitions.SessionHostConfigurationProperties.properties.diskInfo` which is not a top level property

**Comment by almat-msft** [comment]
> secureBootEnabled



> > ```
> >     "secureBootEnabled": {
> > ```
> 
> Replace boolean/switch properties with a more meaningful enum whenever possible.
> 
> A boolean will forever have two valid values (true or false). Enums are always a more flexible and future proof option because they allow additional values to be added in the future in a non-breaking way, e.g. [Enabled, Disabled, Suspended, Deallocated].
> 
> For example two switches "isTypeA" and "isTypeB" should be replaced with one enum "type": [A, B, DefaultType].
> 
> Refers to: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json:7072 in [0110f47](https://github.com/Azure/azure-rest-api-specs/commit/0110f4755725de585b210001d1da98d428423eb0). [](commit_id = [0110f47](https://github.com/Azure/azure-rest-api-specs/commit/0110f4755725de585b210001d1da98d428423eb0), deletion_comment = False)

we are mirroring Compute here, and there is only that much that we can do

**Review by rkmanda — APPROVED** [review]
:shipit:

**Comment by ms-bogdan-rotaru** [comment]
@almat-msft Although you requested to merge the PR, this cannot be done yet as there is one pending requested change which you have to close/address/etc.

**Pull request merged**